### ISSUE TITLE
How to use this with flutter

### ISSUE THREAD

**Issue by guitar9** [MeowType/GraphDart#3]
Do you know how I can use this with flutter?
I want directed graph in my flutter app. 
But I dont know how to use the library.

**Comment by 2A5F** [comment]
https://pub.dev/packages/graph_collection#-installing-tab-

**Comment by guitar9** [comment]
I thought this will show me a visual graph....

**Comment by 2A5F** [comment]
No, this is not a data visualization library, this library provides collections
